recipes/start_jupyter_cm/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "start_jupyter_cm" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 341a8a38e87fa34f8b2188a546028b41cde24715ecfe437cd3ef4268913c63ca
+
+build:
+  number: 0
+  entry_points:
+    - jupyter_context-menu_add = start_jupyter_cm:_add
+    - jupyter_context-menu_remove = start_jupyter_cm:_remove
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [osx]
+
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - start_jupyter_cm
+  commands:
+    - jupyter_context-menu_add --help
+    - jupyter_context-menu_remove --help
+
+about:
+  home: https://github.com/hyperspy/start_jupyter_cm
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: Add entries to start Jupyter from context menu.
+
+extra:
+  recipe-maintainers:
+    - ericpre
+    - francisco-dlp
```

Comment:
@francisco-dlp: can you confirm you are happy to be a maintainer of this recipe?

Reply:
Yes, I am.